etc/db.yaml

```diff
@@ -26,6 +26,9 @@ services:
         class: hanneskod\yaysondb\Engine\FlysystemEngine
         arguments: ['%db.settings%', '@filesystem']
 
+    donor_builder:
```

Comment:
Please move to `container.yaml`. Trying to put only database specific objects in here..